### PR TITLE
refactor(generate-types): add missing DO-NOT-EDIT comment to remaining generated type files

### PR DESCRIPTION
**Pull request by oscard0m**
## 📝 Summary

Add missing `//DO NOT EDIT` comment to remaining generated type files

### CODE REVIEW

File: scripts/generate-types.js [hunk 1]
```diff
@@ -62,6 +62,8 @@ webhooks.forEach(({ name, actions, examples }) => {
 conditionalType.push("never");
 
 const getWebhookPayloadTypeFromEvent = `
+${doNotEditThisFileDisclaimer}
```
Comment (gr2m) [inline scripts/generate-types.js:65]: I think `doNotEditThisFileDisclaimer` needs to be defined in this file for it to work?

File: scripts/update-known-events.js [hunk 2]
```diff
@@ -3,6 +3,9 @@ const Table = require("table-builder");
 const WEBOOOKS = require("@octokit/webhooks-definitions/index.json");
 const { generateFile } = require("./generate-file");
 
+const doNotEditThisFileDisclaimer = `
+// DO NOT EDIT THIS FILE DIRECTLY
```
Comment (gr2m) [inline scripts/update-known-events.js]: I think I would slightly rephrase it
```suggestion
// THIS FILE IS GENERATED - DO NOT EDIT DIRECTLY
```

**Review by gr2m — COMMENTED** [review]
can you generated the `src/generated/*` files so they get updated as part of this PR?

**Comment by oscard0m** [comment]
> can you generated the `src/generated/*` files so they get updated as part of this PR?

sure!

File: scripts/generate-types.js (hunk 1 above)
Comment (oscard0m) [reply scripts/generate-types.js:65]: > I think `doNotEditThisFileDisclaimer` needs to be defined in this file for it to work?

It was already defined. Only missing on some generated files:

https://github.com/octokit/webhooks.js/blob/96c33db3e5a47d0a221afe7d8eca32c6dd58144a/scripts/generate-types.js#L9-L11

**Review by gr2m — APPROVED** [review]
Great!

**Pull request merged**